transforms.py

```diff
@@ -36,12 +36,15 @@ def reset_times(self):
         self._times = {transform: [] for transform in self.transforms}
 
     def extract_times(self):
-        return {
-            re.match(r"(?P<name>.*?)(_?[vV][1,2])?$", type(transform).__name__)[
-                "name"
-            ]: torch.tensor(self._times[transform], dtype=torch.float64).mul_(1e-9)
+        return [
```

Comment:
Changing to a list of tuples here, since the sanitize transform now appears two times in the pipeline.